consensus/src/processes/pruning_proof/build.rs

```diff
@@ -209,6 +207,33 @@ impl PruningProofManager {
         )
     }
 
+    /// Given two hashes (both are assumed to exist in the passed ghostdag store),
+    /// find their latest common ancestor. The latest common ancestor is minimum block
+    /// such that both 'a' and 'b' are either equal to it or in its future.
+    /// This ancestor must also be in the selected parent chain of 'a'.
+    ///
+    /// Additional Notes:
+    /// - 'a' and 'b' are expected to be hashes acquired via block_at_depth.
```

Comment:
`a` is not necessarily a chain block. Also, I don't think the "additional notes" part is really helpful. Maybe change it to say that it returns an error if it doesn't find a common ancestor?